test/e2e/kubectl.go

```diff
@@ -202,6 +202,65 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl expose", func() {
+		It("should check if kubectl expose creates services for rc", func() {
```

Comment:
Nit: shorten to "should create services for rc". Describe() already contains `kubectl expose`